tailwind.config.js

```diff
@@ -21,30 +21,37 @@ module.exports = {
     },
     extend: {
       spacing: {
-        '1px': '0.0625rem',
+        '1px': '0.063rem',
         'm1.5px': '-0.09375rem',
         '1.5px': '0.09375rem',
         '2px': '0.125rem',
         '2.5px': '0.15625rem',
         '3px': '0.188rem',
+        '4px': '0.25rem',
```

Comment:
nit: Tailwind has a lot of defaults, w-1 for 4px, w-2 for 8px, w-3 for 12px and so on. I don't mind to have both variants, but it will make code inconsistent, so in one place we will have p-1, in other place p-4px